guides/common/modules/proc_changing-the-repository-sets-status-for-a-host-in-project.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Changing_the_Repository_Sets_Status_for-a-Host-in_{project-context}_{context}"]
+= Changing the Repository Sets Status for a Host in {Project}
+
+Repository sets show repositories available to each host.
+A host will be able to access content from a given repository if that repository is enabled.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select a host.
+. Select the *Content* tab, then select the *Repository sets* subtab.
+On the tab, there is a set of repositories available to each host with a status of *Enabled* or *Disabled*.
+. You can override the default status by using the action menus on each table row by changing the status to *Override to disabled*, *Override to enabled*, or *Reset to default*.
+. You can also bulk select the checkboxes on each table row, and use the vertical ellipsis icon at the top.
++
+[NOTE]
+====
+For hosts not in the default Content View and lifecycle environment, the *Repository sets* tab shows a toggle group with two options, *Limit to environment* and *Show all*.
+The *Limit to environment* option shows only repositories that are relevant to the host.
+The *Show all* option shows all available repositories including those that may not be in the host's Content View and lifecycle environment.
+On the *Overview* tab, click *Content view details* to view the environment for the host.
+====
+. Select *Show all* or *Limit to environment* to view available repositories.
```

Comment:
The toggle group is _only_ shown for hosts __not__ in the default Content View and lifecycle environment, so it may be confusing to have this step in the procedure. Is it enough to delete `Select *Show all* or *Limit to environment* to view available repositories` and just rely on the note?